### PR TITLE
Liu2021OLTR reivew

### PR DESCRIPTION
**Pull request by sritank**
draft summary

### CODE REVIEW

File: _summaries/liu2021swin_1.md [hunk 1]
```diff
@@ -1,19 +1,72 @@
 ---
 layout: summary
-title: Summary
-paper: {{paper_tag}}
-# Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+title: Large-Scale Long-Tailed Recognition in an Open World
+paper: liu2021swin
```
Comment (mattkelleher) [inline _summaries/liu2021swin_1.md:4]: I believe the paper tag is liu2019large-scale. Might need to move the contents of this file to liu2019large-scale_1.md

Comment (mattkelleher) [reply _summaries/liu2021swin_1.md:4]: also will likely need to rename the images

File: _summaries/liu2021swin_1.md [hunk 2]
```diff
@@ -1,19 +1,72 @@
...
+They use modulated attention to encourage samples from different classes to use different contexts, given by:
+![attention](liu2021swin_1e.png)
+
+A cosine classifier is used as:
+![cosine classifier](liu2021swin_1f.png)
+
+The loss function is a combination of cross entropy and large margin loss between the embeddings and centroids:
+
+![loss](liu2021swin_1g.png)
+
```
Comment (mattkelleher) [inline _summaries/liu2021swin_1.md:45]: very nice breakdown of the math

File: _summaries/liu2021swin_1.md [hunk 3]
```diff
@@ -1,19 +1,72 @@
...
+The model achieves SOTA results on ImageNet-LT, MegaFace and SUN-LT datasets and does very well on Places-LT dataset as shown below:
+
+![performance 1](liu2021swin_1i.png)
+
+![performance 2](liu2021swin_1j.png)
+
 * What interesting variants are explored?
+They explore using different memory features and influence of dataset long tailed-ness. They find that their method is very robust to skewed/imbalanced datsets.
+
+![robustness]](liu2021swin_1k.png)
```
Comment (mattkelleher) [inline _summaries/liu2021swin_1.md:64]: this link to image is not working

**Review by mattkelleher — COMMENTED** [review]
Very solid review. Could mention in the in class discussion the motivation for wanting a system that is capable of this "3 level" or "3 task" classification. What practical applications (if any) do you think this system could be useful for? Personally it seems like while this long tail recognition is indeed the reality of the real world and what we as humans experience I'm not currently convinced there are many practical applications. Maybe as this area develops it will facilitate applications that have not yet been though of.

**Comment by sritank** [comment]
> Very solid review. Could mention in the in class discussion the motivation for wanting a system that is capable of this "3 level" or "3 task" classification. What practical applications (if any) do you think this system could be useful for? Personally it seems like while this long tail recognition is indeed the reality of the real world and what we as humans experience I'm not currently convinced there are many practical applications. Maybe as this area develops it will facilitate applications that have not yet been though of.

Good point. Will make sure to cover these.

**Comment by sritank** [comment]
Fixed all issues

**Comment by sritank** [comment]
@zhaoyue-zephyrus can you please approve this pull request?

**Comment by zhaoyue-zephyrus** [comment]
@sritank Sure. Working on this now.

**Review by zhaoyue-zephyrus — APPROVED** [review]

**Review by zhaoyue-zephyrus — APPROVED** [review]

**Pull request merged**